src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -273,9 +273,7 @@ private RecommendedSymbols GetSymbolsOffOfName(NameSyntax name)
             //
             // Here "stringList.await" is thought of as the return type of a local function.
```

Comment:
This comment should be transferred as a doc comment to `ShouldBeTreatedAsTypeInsteadOfExpression`.